os/arch/arm/src/armv7-m/up_mpu.c

```diff
@@ -355,3 +355,48 @@ void mpu_configure_app_regs(uint32_t *regs, uint32_t region, uintptr_t base, siz
 	regs[2] = regval;
 }
 #endif
+
+/****************************************************************************
+ * Name: mpu_configure_library_region
+ *
+ * Description:
+ *   Configure the mpu region for common library
+ *
```

Comment:
Can we have parameter description for the arguments ?

Reply:
Done